commit 1a41069c0dc054e59cd76fc3d8cf7cc2a78b5e8d
Author: Allen Winter <allen.winter@kdab.com>
Date:   Sat Sep 3 16:56:29 2016 -0400

    CMakeLists.txt, libical.pc.in - fix iculibs (remove full path)
    ISSUE: 227

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0173e1a8..0c903ee5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -145,6 +145,7 @@ endif()
 #  RSCALE info at http://tools.ietf.org/html/rfc7529
 find_package(ICU)
 if(ICU_FOUND)
+  set(ICUUC_LIBS "-licuuc") #for libical.pc
   set(HAVE_LIBICU 1)
   if(ICU_MAJOR_VERSION VERSION_GREATER 50)
     set(HAVE_ICU_DANGI TRUE)
@@ -154,6 +155,7 @@ if(ICU_FOUND)
 endif()
 if(ICU_I18N_FOUND)
   set(HAVE_LIBICU_I18N 1)
+  set(ICUI18N_LIBS "-licui18n") #for libical.pc
 endif()
 
 # MSVC specific definitions
diff --git a/libical.pc.in b/libical.pc.in
index 5c2debd8..8899f430 100644
--- a/libical.pc.in
+++ b/libical.pc.in
@@ -3,10 +3,10 @@ exec_prefix=@exec_prefix@
 libdir=@libdir@
 includedir=@includedir@
 threadslib=@PTHREAD_LIBS@
-iculib=@ICU_LIBRARIES@ @ICU_I18N_LIBRARIES@
+iculibs=@ICUUC_LIBS@ @ICUI18N_LIBS@
 
 Name: libical
 Description: An implementation of basic iCAL protocols
 Version: @LIBICAL_LIB_VERSION_STRING@
-Libs: -L${libdir} -lical -licalss -licalvcal ${threadslib} ${iculib}
+Libs: -L${libdir} -lical -licalss -licalvcal ${threadslib} ${iculibs}
 Cflags: -I${includedir}
